### PR TITLE
Updating relevant feedback items given for the plugin sections in the Getting Started readme

### PR DESCRIPTION
**Pull request by sunithasomanna**
Updated relevant feedback items given for the plugin sections in the Getting Started readme:
- Update parameters for ACI plugin (step 13)
- Specify what is default odim_username and odim_password in the section "Adding a plugin into the Resource Aggregator for ODIM framework"
- Replace hardcoded connectionMethodID in sample payload of aci plugin with a variable {ConnectionMethodId} in the section "Adding a plugin into the Resource Aggregator for ODIM framework".
- To verify the added plugins, need to call 2 APIs. Rephrase to this - " A link of the executed task. Performing a GET operation on this task displays the task monitor associated with this operation in the response header".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
 
         vi kube_deploy_nodes.yaml
 
-13. Update the following parameters in the `kube_deploy_nodes.yaml` file to their corresponding values: 
+13. Specify values for the following parameters in the `kube_deploy_nodes.yaml` file:
```
Comment (sukhwinder20) [inline README.md:203]: cd ODIM --> This has to be removed

Comment (sunithasomanna) [reply README.md:203]: removed

**Review by irappa-pattar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -170,13 +170,7 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
       The Helm package for the Cisco ACI plugin is created in the tgz format.
 
-8. Navigate to the `ODIM` directory.
-
-   ```
-   cd ODIM
-   ```
-
-9. Save the Cisco ACI plugin Docker image on the deployment node at `~/plugins/aciplugin`.
+8. Save the Cisco ACI plugin Docker image on the deployment node at `~/plugins/aciplugin`.
```
Comment (sukhwinder20) [inline README.md:174]: Add 9th step as cd ~/PluginCiscoACI

**Review by sukhwinder20 — CHANGES_REQUESTED** [review]
Kindly address the comment

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md:174]: done

**Review by sukhwinder20 — APPROVED** [review]
LGTM

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**